tests/constraints.txt

```diff
@@ -1,10 +1,9 @@
-importlib-metadata==0.22;python_version < "3.8"
+importlib-metadata==0.22
 packaging==19.0
 pep517==0.9.1
+setuptools==42.0.0; python_version < "3.10"
+setuptools==56.0.0; python_version >= "3.10"
 toml==0.10.0
 tomli==1.0.0
```

Comment:
Do we still need both of these?

Reply:
Not sure honestly, do the tests run with toml?

Reply:
I guess it's a transitive dependency. If an older version of pep517 is installed then it'll require toml, not tomli, so we're testing that an old version of toml works with an old version of pep517?